index.html

```diff
@@ -42,28 +42,28 @@
                     <path fill-rule="evenodd" clip-rule="evenodd" d="M19.69 31.9995C27.51 31.9995 33.2429 29.7956 33.2429 23.8663C33.2429 16.4488 27.51 10.8531 19.69 10.8531C11.87 10.8531 6.26678 16.9693 6.26678 23.8663C6.26678 29.7956 11.87 31.9995 19.69 31.9995ZM19.3825 29.7225C25.1347 29.7225 29.3517 28.1085 29.3517 23.7664C29.3517 18.3345 25.1347 14.2368 19.3825 14.2368C13.6303 14.2368 9.50872 18.7157 9.50872 23.7664C9.50872 28.1085 13.6303 29.7225 19.3825 29.7225Z" fill="white"/>
                 </svg>
            </span>
-           <h1 class="name"><a href="/">판다마켓</a></h1>
+           <h1 class="header-container__logo--title"><a href="/">판다마켓</a></h1>
         </div>
-        <a class="login" href="/signin">로그인</a>
+        <a class="header-container__login-button" href="/signin">로그인</a>
     </header>
-    <main>
-        <section class="panda-banner top-banner">
-            <div>
+    <main class="main-container">
+        <section class="main-container__banner main-container__banner--top-banner">
```

Comment:
```suggestion
        <section class="main-container__banner--top-banner">
```
이런식으로 쓰시면될것같습니다! 👍 
어떤 규칙을 지켜서 쓰던 한번 정한 규칙이나 포맷이 있다면 일관성있게 써주시는게 더 중요합니다
참고해주세요! ㅎㅎ